virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,69 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(sec_dict=None, remote_args=None):
```

Comment:
sec_dict should not be a default value parameter, please delete '=None'.

Reply:
done